packages/api/lambdas/create-reconciliation-report.js

```diff
@@ -857,27 +861,36 @@ async function processRequest(params) {
       // reportType is in ['Inventory', 'Granule Not Found']
       await createReconciliationReport(recReportParams);
     }
-    apiRecord = await reconciliationReportModel.updateStatus({ name: reportRecord.name }, 'Generated');
-    await indexReconciliationReport(esClient, { ...apiRecord, status: 'Generated' }, process.env.ES_INDEX);
+
+    const generatedRecord = {
+      ...reportPgRecord,
+      status: 'Generated',
+    };
+    [reportPgRecord] = await reconciliationReportPgModel.upsert(knex, generatedRecord);
+    reportApiRecord = translatePostgresReconciliationReportToApiReconciliationReport(reportPgRecord); // eslint-disable-line max-len
+    await indexReconciliationReport(esClient, reportApiRecord, process.env.ES_INDEX);
   } catch (error) {
-    log.error(`Error caught in createReconciliationReport creating ${reportType} report ${reportRecordName}. ${error}`);
-    const updates = {
+    log.error(`Error caught in createReconciliationReport creating ${reportType} report ${reportRecordName}. ${error}`); // eslint-disable-line max-len
```

Comment:
Do you have a suggestion on how to break up the line? If I do
```
log.error(
      `Error caught in createReconciliationReport creating ${reportType} report ${reportRecordName}. ${error}`
    );
```
that middle line is still too long. I'm not totally sure, but a brief look and tests suggests that if I put an enter/newline between the backticks, the logging will have that as well.

Reply:
Really? I wonder if your editor settings are more aggressive than the project linter settings? This for me passes linting:

```
log.error(
      `Error caught in createReconciliationReport creating ${reportType} report ${reportRecordName}. ${error}`
    );
```

Reply:
I was failing the linter stage in Bamboo with error messages about max line length of 100. With the indentation, that middle line is still at 111.

Reply:
That's weird. Does it fail if you locally run `npm run lint`? Mine doesn't with the change above. I wonder if there was a change? Or if my config is out of date?

This doesn't require a ton of effort. If it can't be done legally, disabling lint for a couple log lines isn't a deal-breaker.

Reply:
`npm run lint` doesn't flag the line. But if you look in the bamboo logs for any of the builds that failed the audit step you might see error messages like this (this is probably for a different line)
```
build	05-Sep-2024 08:45:41	/cumulus/packages/api/lambdas/create-reconciliation-report.js
build	05-Sep-2024 08:45:41	  870:1  error  This line has a length of 101. Maximum allowed is 100  max-len
build	05-Sep-2024 08:45:41	
build	05-Sep-2024 08:45:41	✖ 1 problem (1 error, 0 warnings)
build	05-Sep-2024 08:45:41	
simple	05-Sep-2024 08:45:41	Failing task since return code of [/data/bamboo/temp/runInDocker3489128750080893134.sh /data/bamboo/bamboo-agent-home/temp/CUM-CBA4068-AUD-11-ScriptBuildTask-5687252********0559282483.sh] was 1 while expected 0
simple	05-Sep-2024 08:45:41	Finished task 'Lint' with result: Failed
```
So even if locally it passes lint, I know it will fail in Bamboo since it has done so before.

Reply:
I'm open to other ways of breaking the line up, but not sure how to do so that is stylistically legit and acceptable.

Reply:
101 characters! So close. Interesting that your local linter doesn't find it. That might be a config issue. I haven't tried locally on my end 🤔 

In any case, we've discussed this longer than a lint error should be discussed 😆. Leaving the disable in there is fine, just something to generally avoid.